templates/events_collection.yaml.epp

```diff
@@ -12,3 +12,6 @@
 <% if $common_events::log_level{ -%>
 "log_level" : "<%= $common_events::log_level %>"
 <% } -%>
+<% if $common_events::log_schedule{ -%>
+"log_rotation" : "<%= $common_events::log_schedule %>"
```

Comment:
Maybe keep to one name for consistency? (maybe replace the log_schedule to all "log_rotation")

Reply:
Done!